backend/pps/sch.py

```diff
@@ -45,5 +50,25 @@ def get_cycle(self, character_stats, caster_tax):
         if ((30 - 2 * short_gcd) % (short_gcd + caster_tax) > 1.5) : 
             result += 6 * (2 * short_gcd + math.ceil((30 - 2 * short_gcd) / (short_gcd + caster_tax)) * (short_gcd + caster_tax)) - 1 * caster_tax
         else:  
-            result += 6 * (2 * short_gcd+math.floor((30-2*short_gcd) / (short_gcd+caster_tax)) * (short_gcd+caster_tax)) - 1 * caster_tax
+            result += 6 * (2 * short_gcd+math.floor((30 - 2 * short_gcd) / (short_gcd + caster_tax)) * (short_gcd + caster_tax)) - 1 * caster_tax
         return result
+
+    # MP consumed over a 3 min cycle (positive = losing mp, negative = gaining mp)
```

Comment:
having positive result represent an mp loss seems counterintuitive.  Please consider multiplying result by -1 when returning instead

Reply:
you can also rename the function to mp_consumption or something because it is unclear what "getting mp" means
also doc strings for functions should be triple quotes below the function declaration, not single line comments above to follow typical python code style

Reply:
renamed